bears/general/LineCountBear.py

```diff
@@ -2,6 +2,8 @@
 from coalib.results.Result import Result
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
+import re
```

Comment:
The order of imports can be preserved, it can be something like : 
```
import re

from coalib.results.Result import 
from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY 


class LineCountBear...
```